tapdb/universe.go

```diff
@@ -264,6 +264,37 @@ func (b *BaseUniverseTree) RegisterIssuance(ctx context.Context,
 	key universe.BaseKey, leaf *universe.MintingLeaf,
 	metaReveal *proof.MetaReveal) (*universe.IssuanceProof, error) {
 
+	var (
+		writeTx BaseUniverseStoreOptions
+
+		err           error
+		issuanceProof *universe.IssuanceProof
+	)
+	dbErr := b.db.ExecTx(ctx, &writeTx, func(dbTx BaseUniverseStore) error {
+		issuanceProof, err = universeRegisterIssuance(
```

Comment:
👍 

this is exactly what I had in mind re db tx pas thru to support re-use elsewhere.

Reply:
Still getting through the diff, but we'll also want to do this for the fetch call as well, since we'll want to fetch the other inclusion proof into the universe tree after that.